tests/ui/test_validation_ui.py

```diff
@@ -30,11 +34,15 @@ class TestUserInterfaceValidation(object):
     @ui
     @tier1
     @black_squad
+    @bugzilla("2155743")
     @polarion_id("OCS-2575")
+    @polarion_id("OCS-4876")
```

Comment:
adding multiple polarion id may cause issues with reporting, please update the new get requests verification step in the existing polarion test case OCS-2575 instead of creating a new test case

Reply:
done https://url.corp.redhat.com/63d2d02